python_modules/dagster/dagster/_core/storage/runs/sql_run_storage.py

```diff
@@ -837,16 +837,29 @@ def get_backfills(
         status: Optional[BulkActionStatus] = None,
         cursor: Optional[str] = None,
         limit: Optional[int] = None,
+        filters: Optional[BulkActionsFilter] = None,
     ) -> Sequence[PartitionBackfill]:
         check.opt_inst_param(status, "status", BulkActionStatus)
-        query = db_select([BulkActionsTable.c.body])
-        if status:
-            query = query.where(BulkActionsTable.c.status == status.value)
+        query = db_select([BulkActionsTable.c.body, BulkActionsTable.c.timestamp])
+        if status or (filters and filters.status):
```

Comment:
unfortunate that we have to do all of this checking around status. I assume we'd have to do a deprecation warning cycle if we wanted to have status filtering be done via the `BulkActionsFilter`?

Reply:
I think so?  I don't have a great sense for how often people are manually calling this from user code with the status arg.

It's not marked as a `@public` method on the instance, but it is exposed on the cloud graphql implementation.  If you wanted to be super-conservative, you could log some volume metrics in Cloud first before deprecating.

Reply:
similarly, should we take this opportunity to make it a `statuses` (plural) filter?

Reply:
Hmm.  Will we need them to render the different tabs in the Runs view?

Reply:
I'm not sure if we need the plural statuses, but `RunsFilter` does statuses and [this internal fn](https://github.com/dagster-io/internal/blob/50492502d7a7ebbec7778fac5ccf08c4ef153b9d/dagster-cloud/python_modules/dagster-cloud-backend/dagster_cloud_backend/storage/host_cloud/run_storage/storage.py#L1223) also uses `statuses` so it could be nice to align